kiwi/system/setup.py

```diff
@@ -276,15 +276,12 @@ def setup_groups(self):
 
         for users_set in self.xml_state.get_users():
             for user in users_set:
-                if user.get_group() and not system_users.group_exists(user.get_group()):
-                    log.info('Adding group %s', user.get_group())
-                    system_users.group_add(user.get_group(), [])
-                if not user.get_secgroups():
-                    continue
-                for group in user.get_secgroups().split(','):
-                    if not system_users.group_exists(group):
-                        log.info('Adding group %s', group)
-                        system_users.group_add(group, [])
+                if user.get_groups():
+                    user_groups = user.get_groups().split(',')
```

Comment:
This is something essentially wrong. I would definitely expect that `user.get_groups()` _already_ splitted AND verified.